code/modules/objectives/experimental_devices.dm

```diff
@@ -38,13 +41,12 @@
 	return ..()
 
 /obj/item/device/reagent_scanner/adv/objective
-	var/datum/cm_objective/retrieve_item/almayer/device/objective
+	var/datum/cm_objective/retrieve_item/device/objective
```

Comment:
same etc for all items, needs to clean up. alternatively, consider using a component to be able to make any item an objective